### ISSUE TITLE
Unable to scroll through the vertical tabs using touchpad/trackpad mouse in laptop

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#30016]
## Description 

Unable to scroll through the vertical tabs using touchpad/trackpad mouse in laptop
## Steps to Reproduce

   1. Clena profile 1.52.x
   2. Enable vertical tabs
   3. Add bunch of sites in vertical tabs
   4. Pin 4 to 5 tabs 
   5. Don't use USB mouse, try to access the vertical tabs using touchpad/trackpad mouse in laptop
   6. Unable to scroll through the vertical tabs

## Actual result:

Unable to scroll through the vertical tabs using touchpad/trackpad mouse in laptop
![vertical tab](https://user-images.githubusercontent.com/38657976/234648766-af0dc51c-2d55-4053-bf94-80af7804ce64.gif)

## Expected result:
Should be able to scroll through the vertical tabs using touchpad/trackpad mouse in laptop

## Reproduces how often: 

Easy
## Brave version (brave://version info)

Brave | 1.52.72 Chromium: 113.0.5672.53 (Official Build) nightly (64-bit)
-- | --
Revision | 12f5dac35d12e8f4e72d7dd11df557ef93bc046f-refs/branch-heads/5672@{#703}
OS | Windows 10 Version 21H2 (Build 19044.2846)

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/qa-team @rebron @sangwoo108

**Comment by sangwoo108** [comment]
Humm, I can scroll it with my Mac and I also tried to connect Win10 via RDP but it looks working. @rebron , how about you?

**Comment by GeetaSarvadnya** [comment]
@sangwoo108 I have not enabled the flag "scrollable tab strip". I can reproduce the issue consistently using touchpad mouse

**Comment by sangwoo108** [comment]
Thanks, I'm lacking reproducible environment. From my experience, some touchpads don't work well with Chromium-based browser. Could we test it with other devices?

**Comment by GeetaSarvadnya** [comment]
@sangwoo108 It's not device specific issue, issue is with the devices which have **touchpads** inbuilt mouse. If i connect USB mouse to my window 10 laptop it works perfectly fine.

**Comment by sangwoo108** [comment]
Yeah, I meant some laptops built-in touchpad doesn't work well with browser's scroll function. So I was wondering if this could be fine on other laptops. I seem to remember I've seen some manufacturers' built-in touchpad and it's driver behaved differently to others.

**Comment by sangwoo108** [comment]
@simonhong , @fallaciousreasoning do you happen to have Windows laptops with touchpad? Unfortunately I don't have any so I'm not able to invesigate this one. Could you help me with this?

**Comment by stephendonner** [comment]
As a datapoint, here are the system specs for my `Windows 10` `Dell` `XPS` laptop

![image](https://user-images.githubusercontent.com/387249/235287508-8b210532-8f6d-44e1-9c3e-2db24fdccabd.png)

It's been working fine for me using 

Brave | 1.53.6 Chromium: 113.0.5672.53 (Official Build) nightly (64-bit)
-- | --
Revision | 12f5dac35d12e8f4e72d7dd11df557ef93bc046f-refs/branch-heads/5672@{#703}
OS | Windows 10 Version 22H2 (Build 19045.2913)

@sangwoo108 let me know if I can help further/provide more info!

**Comment by sangwoo108** [comment]
@stephendonner , thanks for confirming that. If it's really a device(laptop)-dependent problem, I don't feel confident about solving this. I'm lacking expertise here 😭

@GeetaSarvadnya , could you let us know what your laptop is?

**Comment by simonhong** [comment]
> @simonhong , @fallaciousreasoning do you happen to have Windows laptops with touchpad? Unfortunately I don't have any so I'm not able to invesigate this one. Could you help me with this?

Hmm, I also can't scroll on my all devices.

**Comment by fallaciousreasoning** [comment]
Okay, so I have a `Windows 11 Dell XPS 15` (in Windows and in Ubuntu 23.04) and I seem to be able to scroll fine on `1.53.10 Chromium: 113.0.5672.63 (Official Build) nightly (64-bit)` with the touchpad and the scroll wheel on my bluetooth mouse.

![image](https://user-images.githubusercontent.com/7678024/235534106-3cbe6a04-8cfc-4d24-9fef-28cf73086b38.png)

**Comment by sangwoo108** [comment]
@GeetaSarvadnya Thanks for sharing. Could you give us your device name too? like dell xps or lenovo foo, hp elite or something like that.

And I'm also curious what other scrollable UI are like.

1. Task manager - App Menu > more tools > Task manager
<img width="770" alt="image" src="https://user-images.githubusercontent.com/5474642/236216487-d0398c34-dc5c-4004-a9fe-87186398acc1.png">

If you have a bunch of tabs, then you might see some scrollable view on it like below

<img width="671" alt="image" src="https://user-images.githubusercontent.com/5474642/236216684-0a9ca315-af1c-497f-adbf-3401c14eb453.png">

Can you scroll it with your touchpad?


2. Disable vertical tabs and enable scrollable tab strip
<img width="683" alt="image" src="https://user-images.githubusercontent.com/5474642/236217110-14e293f3-9924-4eb5-8a0f-5e6701294b9f.png">

If you have flags set like above and create a lot of tabs until you get some shadow over the leading or  trailing tab, like below.

<img width="441" alt="image" src="https://user-images.githubusercontent.com/5474642/236217475-7bd5c786-af7d-489e-8c37-ad99e0c3710e.png">

Can you scroll the tab strip with the touch pad?

---

If you can scroll 1 and 2 -> then vertical tab strip is the problem.
If you can't -> could be a device-specific problem which upstream has yet to solve.

**Comment by GeetaSarvadnya** [comment]
@sangwoo108 Please find my responses below:

>device name 

**Dell Inspiron 15 5000 Series i7**

>1. Task manager - App Menu > more tools > Task manager

![tabs scrollable in -Task manager](https://user-images.githubusercontent.com/38657976/236247954-405e3e18-d6cf-41d4-b9cd-1fb902976c0f.gif)

>Can you scroll it with your touchpad?

Yes, I can scroll the tabs in task manager using touch pad shown in the GIF file above

>2. Disable vertical tabs and enable scrollable tab strip 

Done shown in the image below
![image](https://user-images.githubusercontent.com/38657976/236249948-abacbcc7-d088-4d33-b1a7-fd6e3806d83f.png)

>Can you scroll the tab strip with the touch pad?

As you mentioned above, I have disabled vertical tabs and enabled #scrollable tabstrip via brave://flags and  created a lot of tabs until you I get some shadow over the leading or trailing tab, like below. **I am unable to scroll through all the tabs using touch pad** - Shown  in the GIF file below

![scrollable tabstrip](https://user-images.githubusercontent.com/38657976/236251993-2707db41-9565-4e60-a01e-180a94339e7b.gif)

**Comment by sangwoo108** [comment]
Thanks @GeetaSarvadnya . It's really helpful!

Could you check one more thing? I'm curious if you can scroll task manager and horizontal scrollable tabstrip with two-finger swipe gesture on touchpad. usually when scroll bar. When we dont' show scrollbar with scrollable view, we expect users to scroll it with the gesture.

**Comment by GeetaSarvadnya** [comment]
@sangwoo108 Please find the twofinger swipe gesture on touchpad on task manger.

![task manager - two finger swipe](https://user-images.githubusercontent.com/38657976/236796437-53fcb063-27aa-4e6a-9b55-6123858857be.gif)

**Comment by sangwoo108** [comment]
Thanks @GeetaSarvadnya . We seem to have the problem only with scrollable tab strip.
